tests/Satooshi/Bundle/CoverallsV1Bundle/Repository/JobsRepositoryTest.php

```diff
@@ -185,72 +137,98 @@ protected function createApiMock($response, $statusCode = 200)
             }
 
             $api
-            ->expects($this->once())
-            ->method('send')
-            ->with()
-            ->will($this->throwException($exception));
+                ->expects($this->once())
+                ->method('send')
+                ->with()
+                ->will($this->throwException($exception));
         }
+    }
 
-        return $api;
+    private function setUpJobsApiWithSendNotCalled($api)
+    {
+        $api
+            ->expects($this->never())
+            ->method('send');
     }
 
-    protected function createResponseMock($statusCode, $reasonPhrase, $body)
+    protected function createApiMockWithRequirementsNotSatisfiedException()
     {
-        $json = is_array($body) ? json_encode($body) : $body;
-
-        $response = $this->getMockBuilder('\GuzzleHttp\Psr7\Response')
-            ->setMethods([
-                'getStatusCode',
-                'getReasonPhrase',
-                'getBody',
-            ])
+        $jobsMethods = $this->setUpJobsApiMethods();
+
+        if (method_exists(__CLASS__, 'createPartialMock')) {
+            $api = $this->createPartialMock('Satooshi\Bundle\CoverallsV1Bundle\Api\Jobs', $jobsMethods);
+        } else {
+            $api = $this->getMockBuilder('Satooshi\Bundle\CoverallsV1Bundle\Api\Jobs')
```

Comment:
Oh, sorry.
Fixd getMock problem and some risky tests.
But 1 risky test may be difficult to fix.